cluster-provision/gocli/build/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM scratch
+FROM alpine:latest
+
+RUN apk add git
```

Comment:
and then we will need to install docker in the image arent we ?
i think it is over kill, i prefer to keep this simple please

btw we can consider using go-git
there is a little issue with it, but theoretically we can live with it (i just less like the fact it is not explicit as the git itself)
https://github.com/go-git/go-git/issues/717
it doesnt detect Rename perfect, it make it looks like Delete and Add
it can work for us because the deleted file will act as deleted file (checks if a matching rule exists, if yes act, else continue without error that it didnt find it, because it is a deleted file)
and the added file will search a rule, act if found, error if not found